File: ecosystems/cocoapods/Tests/FormattingResidueSmoke.swift
import Foundation

@main
struct FormattingResidueSmoke {
    static func main() {
        let actual = FormattingResidue.clean("<p>Hello <span>world</span>…</p><div>Next line</div>")
        let expected = "Hello world...\n\nNext line"
        guard actual == expected else {
            fatalError("clean() = \(actual), want \(expected)")
        }
    }
}

File: ecosystems/maven/src/test/java/pro/aitextcleaner/FormattingResidueSmoke.java
package pro.aitextcleaner;

public final class FormattingResidueSmoke {
    public static void main(String[] args) {
        String actual = FormattingResidue.clean("<p>Hello <span>world</span>…</p><div>Next line</div>");
        String expected = "Hello world...\n\nNext line";
        if (!expected.equals(actual)) {
            throw new AssertionError("clean() = " + actual + ", want " + expected);
        }
    }
}
